### PR TITLE
ci: test plugin when rook deployed in custom namespace

### PR DESCRIPTION
**Pull request by subhamkrai**
this commit adds ci to test plugin when rook deployed
in custom namespace.

Signed-off-by: subhamkrai <srai@redhat.com>

### CODE REVIEW

File: tests/github-action-helper.sh [hunk 1]
```diff
@@ -33,12 +33,35 @@ deploy_rook() {
...
+  wait_for_pod_to_be_ready_state_custom
+  curl https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/storageclass-test.yaml -o storageclass-test.yaml
+  sed -i "s|provisioner: rook-ceph.rbd.csi.ceph.com |provisioner: rook.rbd.csi.ceph.com |g" storageclass-test.yaml
+  deploy_with_custom_ns storageclass-test.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/pvc.yaml
+}
+
+deploy_with_custom_ns(){
+  sed -i "s|rook-ceph # namespace:operator|rook # namespace:operator|g" "$1"
+  sed -i "s|rook-ceph # namespace:cluster|rook # namespace:cluster|g" "$1"
```
Comment (travisn) [inline tests/github-action-helper.sh:60]: How about different namespaces for the operator and cluster, and name them something like `test-cluster` and `test-operator`?

**Review by travisn — CHANGES_REQUESTED** [review]

File: tests/github-action-helper.sh (hunk 1 above)
Comment (subhamkrai) [reply tests/github-action-helper.sh:60]: I think the aim of the PR is to test in different namespaces (from `rook-ceph`) and having different namespaces for operator and cluster PR is still in progress. So, we can wait for the other PR to get merged and I'll update this pr accordingly.

Or we can go ahead with this PR have a new test for different namespaces for the operator and cluster?

Comment (subhamkrai) [reply tests/github-action-helper.sh:60]: I think now these changes are included in #27 only. should we close this?

**Comment by subhamkrai** [comment]
Closing since these changes are included in other PR. Thanks!

**Pull request closed without merging**